AzureData/Source/DocumentClient.swift

```diff
@@ -187,11 +187,17 @@ public class DocumentClient {
     }
     
     // delete
-//    public func delete (_ collection: DocumentCollection, fromDatabase databaseId: String, callback: @escaping (DataResponse) -> ()) {
-//
-//        return self.delete(DocumentCollection.self, at: resourceLocation, callback: callback)
-//    }
-    
+    public func delete (_ collection: DocumentCollection, fromDatabase databaseId: String, callback: @escaping (DataResponse) -> ()) {
+        return self.delete(collectionWithId: collection.id, fromDatabase: databaseId, callback: callback)
```

Comment:
Same here, if we have the `collection` I don't think we should need to pass in the `databaseId`... can we remove this override all together?